.cspell.json

```diff
@@ -52,6 +52,7 @@
     "*.webp",
     "node_modules",
     "coverage",
-    "*.log"
+    "*.log",
+    "test/lock-files/url/**"
```

Comment:
Let's save it in `test/outputs`, we are ignoring this path by default - https://github.com/webpack/eslint-config-webpack/blob/main/ignore-paths.js#L55C8-L55C15, just change a test